### ISSUE TITLE
Trying to get `loadQARefineChain` to use the `input_documents` as context

### ISSUE THREAD

**Issue by spark-ming** [hwchase17/langchainjs#751]
Just noticed the addition of `loadQARefineChain` in `0.0.53`. Thank you!

However, I currently have the code:

```ts
const refineChain = loadQARefineChain(textDavinci);
const result = await refineChain.call({
  input_documents: docs,
  question,
});
```

The is the result I get from calling

```js
{
  output_text: 'Sorry, I cannot answer the question as the provided context information of "[object Promise]" is not related to pre-pay and does not provide enough information to understand the question. Can you please provide more context or clarify the question?'
}
```

Is the context not being serialised correctly?

As a side, I noticed the result is keyed `output_text` instead of `answer` in this chain.

**Comment by RohitMidha23** [comment]
@spark-ming apologies, I seem to have missed this case. Have raised a PR to account for async functionality. You should be able to use RefineChain in your code post that.

**Comment by nfcampos** [comment]
This has been fixed already in 0.0.54
